### ISSUE TITLE
Necessity of this subrepo

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/contrib-too-large#1]
Do we still need this repo? Does anyone know about anyone actually using it?
Seems like it was an approach to combine the contrib sources and the scripts.
I think the current solution with the two repos and the automatic download of only the needed files from the contrib-sources repo is a sufficient solution.
What do you think @OpenMS/core ? In theory we could keep it but it might be cleaner to get rid of when it is certainly unused.

**Comment by hroest** [comment]
this comes from a time when we still had the actual sources inside the repo (in SVN). I think by now this is historic only but maybe @aiche can comment

but given our trouble recently with downloading the sources of different software we could discuss again the merits of this approach, delivering the whole tar.gz files of all software directly in the git repo...

**Comment by aiche** [comment]
> this comes from a time when we still had the actual sources inside the repo (in SVN). I think by now this is historic only but maybe @aiche can comment

correct, this is a 1:1 clone of the SVN contrib repo. shortly after that, we realized that the idea of keeping all the binaries inside the repo wasn't the best idea when using git, so we renamed this repo and setup the new approach of downloading from sourceforge while building.

> but given our trouble recently with downloading the sources of different software we could discuss again the merits of this approach, delivering the whole tar.gz files of all software directly in the git repo.

Just an idea, how about a separate repo containing only the binaries, that is added as shallow submodule (`--depth 1`) to avoid cloning the complete history. While we cannot enforce this, we can at least document it in the build docs.

**Comment by cbielow** [comment]
We have a working solution now and this repo will be deleted on 9th Feb 2017 unless someone objects with a good reason :)
